io.github.purplehorrorrus.Meridius.yml

```
@@ -0,0 +1,38 @@
+app-id: io.github.purplehorrorrus.Meridius
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: meridius
+separate-locales: false
+modules:
+  - name: meridius
+    buildsystem: simple
+    build-commands:
+      - cp -a * /app/bin
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop 
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAT_ID.metainfo.xml
```

```suggestion
      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
```

this is why it can't find it

---

it has nothing to do with `id` vs `app-id` in the manifest as they are equivalent

---

omg. thank you